src/m365/outlook/commands/message/message-remove.ts

```diff
@@ -0,0 +1,142 @@
+import auth from '../../../../Auth.js';
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { accessToken } from '../../../../utils/accessToken.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+import { cli } from '../../../../cli/cli.js';
+import { validation } from '../../../../utils/validation.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  userId?: string;
+  userName?: string;
+  force?: boolean
+}
+
+class OutlookMessageRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MESSAGE_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes a specifc message from a mailbox';
```

Comment:
Typo in specific, same typo in the command description of the docs.

Reply:
copy/paste error from the spec :)

Reply:
Yes, I noticed that indeed, sorry for that 😊